### ISSUE TITLE
Result duplication Error in the diff report

### ISSUE THREAD

**Issue by ChristianLieven** [opencobra/memote#468]
#### Problem description

In a 2-way diff report, when a test in model 2 errors the results of model 1 are displayed instead.

Comparing these two metabolic models (https://github.com/ChristianLieven/memote-m-capsulatus/tree/master/ModelFiles/xml and https://www.biorxiv.org/content/biorxiv/early/2018/06/18/349191.full.pdf) using the `memote diff` function yields duplicated results for the tests `Purely Metabolic Reactions
` and `Purely Metabolic Reactions with Constraints
` although these tests `error` in the snapshot report for the second model.

This may be a separate issue, but ALL tests regarding the biomass reaction seem to be duplicated for the second model, despite their functioning in the snapshot report.

#### Code Sample

```bash
memote report diff iMcBath.xml 349191-9.xml
```

### Context

Please run the following code and paste the output inside the details
block.

```python
import memote
memote.show_versions()
```

<details>
System Information
==================
OS         Darwin
OS-release 16.7.0
Python      3.6.5

Package Versions
================
Jinja2                 2.10
click                   6.7
click-configfile      0.2.3
click-log             0.3.2
cobra                0.12.0
cookiecutter          1.6.0
depinfo               1.3.0
future               0.16.0
gitpython            2.1.10
goodtables            1.0.0
importlib_resources     0.8
memote                0.8.1
numpydoc              0.8.0
pandas               0.23.0
pip                  10.0.1
pygithub               1.39
pylru                 1.1.0
pytest                3.6.1
python-libsbml       5.17.0
requests             2.18.4
ruamel.yaml         0.14.12
setuptools           39.2.0
six                  1.11.0
sqlalchemy            1.2.8
sympy                 1.1.1
travis-encrypt        1.1.0
travispy              0.3.5
wheel                0.31.1
</details>

**Comment by Midnighter** [comment]
I forgot this when looking at the diff report PR. Take a look at [the following lines](https://github.com/opencobra/memote/blob/cb2265f46fe20478ffb12774704562ccda3f26b5/memote/suite/cli/runner.py#L291) in the `memote history` command. Basically, each test must run in a child process. This is due to pytest setup and I haven't found another way around it.

**Comment by ChristianLieven** [comment]
Fixed in #486. Please reopen this issue if there are any remaining problems.
